src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  position: relative;
+  max-width: 1200px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: auto;
+  padding: 16px;
+  align-items: center;
+}
+
+.logo {
+  width: 20px;
+  height: 40px;
+  margin-left: 40px;
+}
+
+.logo img {
+  max-height: 60px;
+  display: block;
+}
+
+.navigation {
+  position: relative;
+}
+
+nav ul {
```

Comment:
do not add styles using tag selectors - use class selectors instead
do not use combined (chained) selector without necessarily - you can just add class to the `ul` tag in HTML and use it to set style - fix all similar cases